utils.js

```diff
@@ -4255,6 +4256,109 @@ export async function parseSetupPyFile(setupPyData) {
   return await parseReqFile(lines.join("\n"), false);
 }
 
+/**
+* pixiMapper used with a map on pixi packages list.
+* the pixi list contains the following information e.g.
+* {kind: conda
+*  name: _libgcc_mutex
+*  version: '0.1'
+*  build: conda_forge
+*  subdir: linux-64
+*  url: https://conda.anaconda.org/conda-forge/linux-64/_libgcc_mutex-0.1-conda_forge.tar.bz2
+*  sha256: fe51de6107f9edc7aa4f786a70f4a883943bc9d39b3bb7307c04c41410990726
+*  md5: d7c89558ba9fa0495403155b64376d81
+*  license: None
+*  size: 2562
+*  timestamp: 1578324546067
+* }
+* We create the purl using the following logic:
+* "purl": "pkg:{kind}/{name}@{version}"
```

Comment:
@aryan-rajoria For entries that include build, we will want to include that at the end of the version `pkg:{kind}/{name}@{version}-{build}` to be specific.

Reply:
added along with os:
`pkg:{kind}/{name}@{version}-{build}?os={subdir}`